docs/README.md

```diff
@@ -0,0 +1,328 @@
+
+### Model danych
+
+Model danych jest dostępny w dokumentacji API.
+
+Wyróżniamy następujące elementy:
+
+* Dictionary - dane na temat słowników danych statystycznych
+* Case - dane na temat sprawy
+* Institution - informacje o instytucjach
+* Tag - dodatkowa konfiguracja prezentacji znaczników
+* Collection - kolekcje umożliwiające dokonanie wyboru prezentowanych spraw poprzez wskazanie kryterium ich wyszukiwania
+* Letter - dane na temat listów przypisanych do sprawy / nowych
+* File - dane na temat plików powiązanych z listem
+* Channel - słownik zawierający konfiguracje kanałów komunikacji
+* Note - notatka wprowadzona przez użytkownika
+* Event - dane na temat wydarzenia terminarza
+* User - dane na temat użytkowników systemu
+* Description - słownik opisów listów
+* AdministrativeUnit - jednostki podziału terytorialnego
+
+### Model uprawnień
+
+System wyróżnia następujące uprawnienia:
+
+* Dictionary.change
+* Dictionary.view
+* Dictionary.delete
+* Dictionary.create
+* Case.change
+* Case.view
+* Case.delete
+* Case.create
+* Institution.change
+* Institution.view
+* Institution.delete
+* Institution.create
+* Tag.change
+* Tag.view
+* Tag.delete
+* Tag.create
+* Collection.change_own
+* Collection.change_all
+* Collection.view
+* Collection.delete
+* Collection.create
+* Letter.change_own
+* Letter.change_all
+* Letter.view
+* Letter.delete
+* Letter.create
+* Channel.change
+* Channel.view
+* Channel.delete
+* Channel.create
+* Note.change_own
+* Note.change_all
+* Note.view
+* Note.delete
+* Note.create
+* Event.change_own
+* Event.change_all
+* Event.view
+* Event.delete
+* Event.create
+* User.change_own
+* User.change_all
+* User.view
+* User.delete
+* User.create
+* Description.change
+* Description.view
+* Description.delete
+* Description.create
+* AdministrativeUnit.change
+* AdministrativeUnit.view
+* AdministrativeUnit.delete
+* AdministrativeUnit.create
+
+Klasy uprawnień mają następujące znaczenie:
+
+* change - edycja wszystkich elementów danego typu
+* view - wyświetlanie wszystkich elementów danego typu
+* delete - usunięcie wszystkich elementów danego typu
+* change_own - edycja elementów danego typu utworzonych przez tego samego użytkownika
+* change_all - edycja elementów danego typu utworzonych przez dowolnego użytkownika
+
+W celu kontroli dostępu wykorzystywane jest:
+
+* uwierzytelnianie  poprzez GSuite - oznaczone w dokumentacji API jako ```sessionAuth```
+* uwierzytelnianie hasłem - wyłącznie w celach administracyjnych, tożsame z oznaczeniem w API jako ```sessionAuth```
+* uwierzytelnianie kodem JWT - oznaczone w dokumentacji API jako ```bearerAuth```.
+
+Dokumentacja API pomija uwierzytelnianie w ten sposób jako standardowy protokół.
+
+### Model powiadomień
+
+Model powiadomień zostanie zdefiniowany po zrealizowaniu podstawowych elementów edycji.
+
+## Panel Zarządzania
+
+### Założenia
+
+* framework - Vue/React
```

Comment:
>> "utworzenie projektu Vue / React i jego wstępna konfiguracja"
> 
> Trzeba by na początek zdecydować, czy Vue, czy React. Obu używam i lubię, ale jeśli mogę coś zasugerować, to React, ponieważ:
> * zero problemów z Typescriptem (przy samym Vue też, ale już z Vuexem trzeba trochę kombinować - a gdzieś stan trzeba trzymać),
> * dzięki ContextAPI w niedużych aplikacjach reaktowych można obecnie w ogóle uniknąć użycia dodatkowej biblioteki store'a (Vuex / Redux),
> * większa szansa znaleźć wolontariusza, który ogarnia Reacta niż Vue,
> * przy dobrze zrobionej architekturze stosunkowo prosto będzie przerobić to w przyszłości na aplikację mobilną z użyciem React Native.

Reply:
Osobiście preferuje Vue, gdyż mam z nim doświadczenie. Wiem jednak, że nie będę miał możliwości znaczącego zaangażowania w projekt, więc pozostawiam to do dyspozycji wykonawcy. Myślę, że obie technologie długofalowo są możliwe w utrzymaniu, a to jest dla mnie istotne.